### DISCUSSION TITLE
Video Downloadhelper deactivates and no longer works.

### DISCUSSION THREAD

**Discussion by Novyvon** [aclap-dev/video-downloadhelper#418] [Help and Q&A]
### Your question

Hello,
Video DownloadHelper has been deactivated and then has not been working for me for several weeks.
I uninstalled, re-installed ... analyzed, cleaned ... After several attempts, it ends up restarting. But only a few days, then the problems come back. I have redone this re-installing cycle to many times for weeks, but nothing helps. After each re-installation, it works for a few days, then deactivates again.

Symptoms :
When I select a video to download, a message tells me that it is on the waiting list, then nothing happens.
In the options, the different files in which download no longer appear. Unable to select where to download. When I click on the arrow allowing you to select a folder, nothing develops.

Thank you for your help.

_________________________

⊖
Video DownloadHelper 8.2.2.8 (stable)
Video DownloadHelper
Version 8.2.2.8 (stable)
Langue du navigateur: fr
Généré pour la production
Généré le 2024-03-18
Options de génération: browser=firefox, linuxlic=false, noyt=false, target=mozilla
⊖
Plateforme
Plateforme Win x86-64
Navigateur Mozilla Firefox 125.0.1

### Addon & CoApp version and details

⊖
Video DownloadHelper 8.2.2.8 (stable)
Video DownloadHelper
Version 8.2.2.8 (stable)
Langue du navigateur: fr
Généré pour la production
Généré le 2024-03-18
Options de génération: browser=firefox, linuxlic=false, noyt=false, target=mozilla
⊖
Plateforme
Plateforme Win x86-64
Navigateur Mozilla Firefox 125.0.1

**Comment by Mattmjs** [comment]
Have you checked anti virus and firewall are not blocking the companion app ?
